pkg/disk/cloud.go

```diff
@@ -980,14 +980,13 @@ func getDiskType(diskVol *diskVolumeArgs) ([]string, []string, error) {
 		log.Log.Infof("CreateVolume:: node support disk types: %v, nodeSelected: %v", nodeSupportDiskType, diskVol.NodeSelected)
 	}
 
-cusDiskType:
 	provisionDiskTypes := []string{}
 	allTypes := deleteEmpty(strings.Split(diskVol.Type, ","))
 	if len(nodeSupportDiskType) != 0 {
 		provisionDiskTypes = intersect(nodeSupportDiskType, allTypes)
 		if len(provisionDiskTypes) == 0 {
 			log.Log.Errorf("CreateVolume:: node(%s) support type: [%v] is incompatible with provision disk type: [%s]", diskVol.NodeSelected, nodeSupportDiskType, allTypes)
-			return nil, nil, status.Errorf(codes.InvalidArgument, "CreateVolume:: node support type: [%v] is incompatible with provision disk type: [%s]", nodeSupportDiskType, allTypes)
+			return nil, nil, status.Errorf(codes.ResourceExhausted, "CreateVolume:: node support type: [%v] is incompatible with provision disk type: [%s]", nodeSupportDiskType, allTypes)
```

Comment:
We can only reach this line if the CSI plugin is just installed, and scheduling is performed before `UpdateNode()` finished. Is that correct?

If the above is true, we can eliminate this kind of racing by using topology to constraint disk type.

Reply:
No, there are other scenarios. We'll leave it as it is for now.